upgrade-1.x-to-2.x.sh

```diff
@@ -76,6 +88,14 @@ else
     esac
 fi
 
+# Find boot path
+boot_path=$(findmnt -n --raw --evaluate --output=target /dev/mmcblk0p1)
+if [ -z "${boot_path}" ]; then
+    log ERROR "Cannot find where is /dev/mmcblk0p1 mounted"
```

Comment:
@agherzan sure. Currently there are probably more assumptions as well because of targeting Pi and BB, but one less assumption is better. Will update.